gqcp/include/FockSpace/Configuration.hpp

```diff
@@ -30,6 +30,14 @@ namespace GQCP {
 struct Configuration {
     ONV onv_alpha;
     ONV onv_beta;
+
+    /**
+     *  @param delimiter        the delimiter between bits
+     * 
+     *  @return a compact representation of the configuration as string:
+     *   e.g: 1100 | 1010 => 2110
+     */
+    std::string asCompactString(const std::string& delimiter = "") const;
```

Comment:
Maybe there's a better word for 'compact', here.

```suggestion
    std::string spinSummedRepresentation(const std::string& delimiter = "") const;
```